packages/cli/src/commands/init/utils/repo.ts

```diff
@@ -96,25 +141,43 @@ export const configureGitSetup = (
           task.skip('Checkout out the branch failed.');
           throw new Error(err);
         })
+    },
+    {
+      title: 'Remove .git',
```

Comment:
Both of these step should be done in just one step i.e. `Initializing git repository`

Reply:
Made into one function.